services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/models/IDCount.java

```diff
@@ -0,0 +1,15 @@
+package us.dot.its.jpo.ode.api.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.EqualsAndHashCode;
+
+@ToString
+@Setter
+@EqualsAndHashCode
+@Getter
+public class IDCount {
+    public String id;
+    public double count;
```

Comment:
chore: encapsulate fields and prevent direct access
```suggestion
    private String id;
    private double count;
```